frank-doc-doclet/src/test/resources/doc/hierarchy-digester-rules.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<digester-rules>
+	<rule pattern="*/param" registerMethod="addParameter"/>
```

Comment:
Dit is een tab en geen 2-space indenting zoals in je .editorconfig staat. Wat moet het worden? ;-)

Reply:
Elk bestand is iedere keer anders wat super vervelend is, vandaar die regel in de editorconfig. Aangepast!